django-stubs/test/client.pyi

```diff
@@ -128,25 +131,136 @@ class Client(ClientMixin, _RequestFactory[_MonkeyPatchedWSGIResponse]):
     ) -> None: ...
     # Silence type warnings, since this class overrides arguments and return types in an unsafe manner.
     def request(self, **request: Any) -> _MonkeyPatchedWSGIResponse: ...
-    def get(  # type: ignore
+    @overload  # type: ignore
```

Comment:
whilst you're modifying this, can use the specific code?

Reply:
@adamchainz done, thank you for pointing this out!

Reply:
Thanks, I’d like us to require specific codes throughout at some point.